src/basic/unit-file.c

```diff
@@ -284,7 +284,7 @@ int unit_file_build_name_map(
                         continue;
                 }
 
-                FOREACH_DIRENT_ALL(de, d, log_warning_errno(errno, "Failed to read \"%s\", ignoring: %m", *dir)) {
+                FOREACH_DIRENT(de, d, log_warning_errno(errno, "Failed to read \"%s\", ignoring: %m", *dir)) {
```

Comment:
note that this means' youll suddenly add ".#foobar.d" to the the map you maintain. I doubt that's really a good idea, no?

Reply:
"Add"? FOREACH_DIRENT should skip it, no?

Reply:
oops, my brain was inverted. sorry!